tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/cookiesBanner.ts

```diff
@@ -0,0 +1,32 @@
+import { Logger } from "../common/logger";
+
+class CookiesBanner {
+    public accept(): this {
+        cy.getByTestId("banner-accept-analytics-cookies-button").click();
+
+        return this;
+    }
+
+    public reject(): this {
+        cy.getByTestId("banner-reject-analytics-cookies-button").click();
+
+        return this;
+    }
+
+    public viewCookies(): this {
+
+        cy.getByTestId("banner-view-cookies").click();
+
+        return this;
+    }
+
+    public notVisible(): this {
```

Comment:
```suggestion
    public isNotVisible(): this {
```
Makes it more clear what the purpose of this function is ie is it checking if the banner is not visible or is it making the banner not visible?